server/routes.js

```diff
@@ -9,6 +9,7 @@
     require('./routes/files')(router);
     require('./routes/error')(router);
     require('./routes/upload')(router);
+    require('./routes/downloadRouter')(router);
```

Comment:
These files are in different directives/namespaces. But I agree with you, this may confuse.